doc/guides/decoding/decoder_options.rst

```diff
@@ -91,6 +91,19 @@ processing applied after decoding to a NumPy :class:`~numpy.ndarray`:
 Miscellaneous Options
 =====================
 
+The following options may be used with both native (uncompressed) and encapsulated
+(compressed) transfer syntaxes when decoding to a NumPy :class:`~numpy.ndarray`:
+
+* `apply_shift_correction`: :class:`bool` - if ``True`` (default) and *Bits Stored*
+  doesn't equal *Bits Allocated* then apply bit-shifting operations to correct
+  for any misinterpretation due to the unused bits. For example, if *Bits Stored*
+  is 5, *Bits Allocated* is 8 and *Pixel Representation* is 1 then the 3 most
```

Comment:
Maybe use a more realistic example, like *BitsAllocated* is 16 and *BitsStored* is 12.

Reply:
I think it's easier to understand as it is, since I can use a single-byte binary instead of a 2-byte one.

Reply:
Fair enough.